### PR TITLE
Pods 3043

### PR DESCRIPTION
**Pull request by richard-poole**

### CODE REVIEW

File: app/views/register/trustees/trusteeKind.scala.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
         @components.input_radio(
             field = form("value"),
-            legend = messages("messages__trusteeKind__legend"),
+            legend = "",
```
Comment (vinnicombe) [inline app/views/register/trustees/trusteeKind.scala.html]: Why the empty legend?

Comment (richard-poole) [reply app/views/register/trustees/trusteeKind.scala.html]: after a discussion earlier it was agreed that the legend will be incorporated into the h1 at a later date so for now the legend should be empty

Comment (vinnicombe) [reply app/views/register/trustees/trusteeKind.scala.html]: This still creates an empty span element which is invalid HTML. I agree with your comment, legend should be incorporated into the H1 where there is no other question/lede text on the page, but until that is implemented legend should still be populated.

Comment (richard-poole) [reply app/views/register/trustees/trusteeKind.scala.html]: legend updated as visually hidden text

Comment (grantrobson) [reply app/views/register/trustees/trusteeKind.scala.html]: Checked - text reinstated.

**Pull request merged**